share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
Sure! That's a better idea. I'll update it shortly. :+1: 

> Sorry for the extra work...

No issues at all. After trying that template only, we were able to know that it won't work well in this case. :smile: 

However, I would still want someone to go back and check why `info` template is causing this issue when the title is too long? Someone in future may ignore this thing and go on with `info` template hence spoiling user experience on DDG.